apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/SnifferConfigInitializer.java

```diff
@@ -41,12 +41,14 @@
  */
 public class SnifferConfigInitializer {
     private static final ILog logger = LogManager.getLogger(SnifferConfigInitializer.class);
-    private static String CONFIG_FILE_NAME = "/config/agent.config";
+    private static String SPECIFIED_CONFIG_PATH = "sw.specified_config_path";
```

Comment:
Why use `sw.` as prefix? All other settings are using `#ENV_KEY_PREFIX` as prefix. Could you make this consistent?

Reply:
The reason i use sw as prefix is that #ENV_KEY_PREFIX is use to override the properties in agent .config...so i just wonder whether it will make some trouble when i use #ENV_KEY_PREFIX as prefix or not...if you think the prefix should be same, i would like to make it consistent..

Reply:
I think it is harmless. But got your point about misleading others. Let's name id `skywalking_config=` Good for that?

Reply:
i have fixed all problems that we just talking about..